README.rst

```diff
@@ -44,7 +44,7 @@ Here's how to build a simple grammar:
 
 You can have forward references and even right recursion; it's all taken care
 of by the grammar compiler. The first rule is taken to be the default start
-symbol, but you can override that.
+symbol, but you can start at any other rule as well.
```

Comment:
```suggestion
symbol, but the grammar will match any other rule as well.
```

Reply:
To my ear, it sounds like that means the grammar will automatically try other rules if the starting one fails. But that's not the case.

Reply:
```suggestion
symbol, but you can use another rule as the start symbol by accessing it directly from the grammar (e.g. `grammar['text'].parse(...)`).
```

Might be a little verbose.

Reply:
Sounds alright to me. I don't think there's a perfect place for it in the current, scant readme. It interrupts the flow of the easy case, but someone was wondering about it, so it's better to mention it than not. Just replace the single backtick with doubles (because that's ReStructuredText for "code"; single backticks are Markdown), and we're ready to fly.